buildSrc/src/testKit/thirdPartyAudit/sample_jars/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'java'
+}
+repositories {
+    mavenCentral()
+}
+dependencies {
+    compile 'org.apache.logging.log4j:log4j-core:2.11.1'
+}
+
+clean.enabled = false
```

Comment:
This seems weird. At least it is worth a comment about why we need it. Do we need it?